packages/gerber-parser/test/gerber-parser_gerber_test.js

```diff
@@ -488,7 +488,16 @@ describe('gerber parser with gerber files', function () {
             y2: 6,
             rot: 7
           },
-          {type: 'vect', exp: exp, width: 2, x1: 3, y1: 4, x2: 5, y2: 6, rot: 7}
+          {
```

Comment:
Apparently this had been changed without running `yarn run format` (best guess). Will explore adding husky or something to add a hook